.env.template

```diff
@@ -4,3 +4,8 @@ FLASK_ENV=development
 
 # Change the following values for local development.
 SECRET_KEY=secret-key
+TRELLO_TODO='610beb5a99354405b1f4ec6f'
+TRELLO_DOING='610beb5a99354405b1f4ec70'
+TRELLO_DONE='610beb5a99354405b1f4ec71'
```

Comment:
Not strictly important here but just a heads up as it might bite you later, the standard format for this .env file won't expect quotes around strings. Flask's dotenv implementation is unphased and handles this fine, but you might see issues when it comes to Docker if you leave the quotes